doc/LIMITS.md

```diff
@@ -223,24 +269,26 @@ const node = await createLibp2p({
      * all connection limits
      */
     allow: [
-      '/ip4/43.123.5.23/tcp/3984',
-      '/ip4/234.243.64.2',
-      '/ip4/52.55',
+      "/ip4/43.123.5.23/tcp/3984",
```

Comment:
This should  follow the same linting rules as the main codebase, so use single quotes.